string model_name   # the name of the model to predict
string instruction  # the instruction sended to model for predicton
string unnorm_key   # the dataset of the scene

---

bool store_ret  # the result of storing the new action